README.md

```diff
@@ -104,6 +104,38 @@ make
 
 At this point, the Module libraries are ready in `build/.libs`.
 
-### Using Mesos Modules
+### Deploying Serenity Module
 
-See [Mesos Modules](http://mesos.apache.org/documentation/latest/modules/).
+Create json and point there built serenity shared library:
```

Comment:
This sounds a little awkward to me.  Suggest rewording to read: "Create a JSON file that describes the shared library and its parameters to the Mesos slave process:"